tsconfig.base.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Why it has been split into a separate file if we are using the same configuration for specs, app, and e2e? Shouldn't we just skip `extends` and reference additional options by `references` directly using `tsconfig.json`?

On the other side, if `references` are not inheriting main file properties, `extends` makes sense here. This issue probably confirms that - https://github.com/microsoft/TypeScript/issues/27098 but I'm not 100% sure. Could you explain that a bit?

Reply:
This file was created during the migration process to the newer version of Angular - I didn't touch it. I guess the framework infrastructure changed a bit. The comment that was automatically added to the `tsconfig.json` file seems to confirm it - https://github.com/ckeditor/ckeditor4-angular/pull/124/files#diff-e5e546dd2eb0351f813d63d1b39dbc48R1-R6.